playbooks/vars/main.yml

```diff
@@ -265,3 +265,37 @@ maas_holland_enabled: false
 #                            in the galera containers (RPC-O newton and onward).
 #
 maas_holland_venv_enabled: false
+
+#
+# nova_service_check_protocol: Protocol to use when authenticating for nova
+#                              service checks. Valid values are "http" or
+#                              "https".
+# TODO Should this be moved to the openstack vars file?
```

Comment:
it makes sense that this is in the main vars as it will be shared across various services and it's nice to only have to modify or look for these vars in a single place.

Reply:
ok, will move - I wasn't sure whether those vars files actually got used, so I stuck it in a place where I knew the templates would find it.

Reply:
oh, never mind - I'll remove the TODO